decidim-participatory_processes/spec/system/filter_participatory_processes_spec.rb

```diff
@@ -259,23 +253,23 @@
 
         context "and choosing 'past' processes" do
           before do
-            skip "REDESIGN PENDING - Filter integration in processes index pending"
-
-            within ".order-by__tabs" do
-              click_link "Past"
+            within "#panel-dropdown-menu-date" do
+              click_filter_item "Past"
             end
             sleep 2
           end
 
           it "lists past processes" do
             within "#processes-grid h2" do
-              expect(page).to have_content("2 PAST PROCESSES")
+              expect(page).to have_content("2 past processes")
             end
           end
 
           it "only shows process types with past processes" do
-            within "#process-type-filter" do
-              click_button "All types"
+            skip "REDESIGN_PENDING - This feature is deprecated with current filters. Remove if it's correct"
```

Comment:
Can we take a decission regarding this ?

Reply:
Yes, I think that these test should be removed. The filter options don't change dynamically with the results of the current query, so I think that these tests no longer make sense. I've added a commit removing them.